bin/compile

```diff
@@ -21,10 +21,14 @@ source "$BUILDPACK_DIR/bin/common.sh"
 topic "Installing Zenaton Agent"
 export SKIP_START=1
 curl https://install.zenaton.com/ | sh
-cp -R "$HOME/.zenaton" "$BUILD_DIR/.zenaton"
+cp -R "$HOME/.zenaton" "$BUILD_DIR"
 
-# Install the Zenaton Agent runner
-topic "Installing Zenaton Agent runner"
+# Create zenaton logs file
+echo "Creating zenaton.out & zenaton.err"
```

Comment:
You can use the `topic` function instead which will display text in the same way as the rest of the heroku stack.

```suggestion
topic "Creating zenaton.out & zenaton.err files"
```